config/example-menu.cfg

```diff
@@ -3,153 +3,116 @@
+#   Position where an item needs to be inserted in list
+#   By default the item is added at the end. This parameter is optional.
 #scroll:
-#   This attribute accepts static boolean value. You can use it together with 'width'.
-#   When this is enabled then names longer than width are scrolled back and forth.
-#   The default is disabled. This parameter is optional.
-#enable:
-#   This attribute accepts static boolean values and parameters (converted to boolean).
-#   It accepts multiple logical expressions. Values separated by comma will return True if all elements are true.
-#   Values on different lines will return True if any element is true.
-#   You can use logical negation by using character ! as parameter prefix.
-#parameter:
-#   This attribute accepts float values or special variables. Multiple values are delimited by comma.
-#   All available parameter variables can be listed by 'MENU DO=dump' gcode, menu itself must be running.
-#   This value is available for output formatting as {0}..{n} Where n is count of parameters.
-#transform:
-#   This attribute allows to transform parameters value to something else.
-#   More than one transformation can be added. Each transformation must be on separate line.
-#   These transformed values are available for output formatting as {n+1}..{x}
-#   Where n is count of parameters and x is count of transformations.
-#   In order to transform the value of a particular parameter, you must add
-#   an parameter index as prefix. Like this "transform: 1.choose('OFF','ON')"
-#   If the index is not set then the default index 0 is used.
-#
-#   map(fromLow,fromHigh,toLow,toHigh) - interpolate re-maps a parameter value from one range to another.
-#   Output value type is taken from toHigh. It can be int or float.
-#
-#   choose(e1,e2) - boolean chooser, converts the value of the parameter to the boolean type (0 and 1),
-#   and selects the corresponding value by the index from the list.
-#
-#   choose(e1,e2,...) - int chooser, converts the value of the parameter to the int type
-#   and selects the corresponding value by the index from the list.
-#
-#   choose({key:value,..}) - special dictionary chooser, parameter value cast type by first key type.
-#   Selects the corresponding value by the key from the dictionary.
-#
-#   int(), float(), bool(), str(), abs(), bin(), hex(), oct(), days(), hours(), minutes(), seconds()
-#   These will convert parameter value to the special form.
-#   int,float,bool,str,abs,bin,hex and oct are python functions.
-#   days,hours,minutes,seconds will convert parameter value (it's taken as seconds) to time specific value
-#
-#   scale(xx) - Multiplies parameter value by this xx. Pure interger or float value is excpected.
-
+#   This attribute accepts True or False. When this is enabled then
+#   names longer than display size are scrolled back and forth.
+#   The default is False. This parameter is optional.
+#script_<name>:
+#   Custom script - Evaluated as a template.
```

Comment:
As before, I don't think the menu should add any additional scripting capabilities.  At least for now.